internal/saucecloud/espresso.go

```diff
@@ -80,6 +88,35 @@ func (r *EspressoRunner) runSuites(appFileID string, testAppFileID string) bool
 	return r.collectResults(r.Project.Artifacts.Download, results, jobsCount)
 }
 
+func (r *EspressoRunner) dryRun(appFileID, testAppFileID string) error {
+	log.Warn().Msg("Running tests in dry run mode.")
+	tmpDir, err := os.MkdirTemp("./", "sauce-app-payload-*")
+	if err != nil {
+		return err
+	}
+	for _, s := range r.Project.Suites {
+		for _, c := range enumerateDevicesAndEmulators(s.Devices, s.Emulators) {
+			opts := r.createJobOpts(s, appFileID, testAppFileID, c)
+			log.Info().Msgf("The [%s] suite would run on %s %s %s.", opts.DisplayName, c.name, c.platformName, c.platformVersion)
+			b, err := json.Marshal(opts)
+			if err != nil {
+				return err
+			}
+			fileName := filepath.Join(tmpDir, fmt.Sprintf("%s_%s_%s_%s.json", unifyStr(opts.DisplayName), unifyStr(c.name), unifyStr(c.platformName), c.platformVersion))
```

Comment:
Imho, I don't think we need those files at all. It's essentially just the json payload that'd be sent to test-composer. Nothing a regular user would care about (or make potentially sense out of it). Logging out the detailed suite information like you have on line 100 is enough 👍

Reply:
I see.

Reply:
Removed generating file part.